.github/workflows/main.yml

```diff
@@ -134,8 +134,9 @@ jobs:
         env:
           DRIVER_NAME: docker
         run: |
-          for roledir in roles/*/molecule; do
-            pushd $(dirname $roledir)
+          roles=$(git diff develop --stat | grep roles | cut -d "/" -f 2 | uniq)
```

Comment:
This means it won't run after a merge (as the diff is empty in that case) and do the wrong thing for PRs against non-develop branches (you could use [GITHUB_BASE_REF](https://docs.github.com/en/actions/reference/environment-variables) to avoid that).

Reply:
Do we run the tests after a merge?

Reply:
yes:
https://github.com/theforeman/foreman-operations-collection/blob/9dce0978cb7ebfe2f49e835c31f9de0da0fc8097/.github/workflows/main.yml#L3

(the `push` part is what would run on merge)

Reply:
Hrmm, well I am gonna drop this for now. I am also thinking about how we may use other roles in our tests for other roles. We should probably ensure those dont break.